espnet2/bin/s2t_inference.py

```diff
@@ -189,6 +190,10 @@ def __init__(
         lang_sym: str = "<eng>",
         task_sym: str = "<asr>",
         predict_time: bool = False,
+        partial_ar: bool = False,
+        threshold_probability: float = 0.99,
+        max_seq_len: int = 5,
+        max_mask_parallel: int = 5,
```

Comment:
Can you move these new arguments before this line? In the current design, the arguments below that line can be overwritten in `__call__`, but these new arguments can only be provided in `__init__`.
```
# default values that can be overwritten in __call__
```